src/compat.js

```diff
@@ -14,6 +14,14 @@ module.exports.tap = (tappable, hook, name, plugin) => (
   : tappable.plugin(hook, plugin)
 );
 
+module.exports.tapHtml = (tappable, name, plugin) => {
+  const HtmlWebpackPlugin = require('html-webpack-plugin');
```

Comment:
It's using `"html-webpack-plugin": "^3.2.0"` from `package.json` here.
`HtmlWebpackPlugin.getHooks` will always be undefined.

Or at least that's what I'm noticing testing it locally.
Makes sense it's undefined tho, the version with getHooks is `html-webpack-plugin@next`.

Reply:
Unless I misunderstood, `html-webpack-plugin` ist a dev-dependency and should not be installed if you consume `webapp-webpack-plugin` as a dependency, meaning it should pick whichever version you have in your package.json.

Reply:
I think you're right, my understanding of npm dependencies is lacking at best.